include/utils.h

```diff
@@ -39,156 +39,148 @@ class CurlHandle {
 
-	static unsigned int get_random_value(unsigned int max);
+	unsigned int get_random_value(unsigned int max);
 
-	static std::string quote_if_necessary(const std::string& str);
+	std::string quote_if_necessary(const std::string& str);
 
-	static void set_common_curl_options(CURL* handle, ConfigContainer* cfg);
+	void set_common_curl_options(CURL* handle, ConfigContainer* cfg);
 
-	static curl_proxytype get_proxy_type(const std::string& type);
-	static unsigned long get_auth_method(const std::string& type);
+	curl_proxytype get_proxy_type(const std::string& type);
+	unsigned long get_auth_method(const std::string& type);
 
-	static bool is_special_url(const std::string& url);
-	static bool is_http_url(const std::string& url);
-	static bool is_query_url(const std::string& url);
-	static bool is_filter_url(const std::string& url);
-	static bool is_exec_url(const std::string& url);
+	bool is_special_url(const std::string& url);
+	bool is_http_url(const std::string& url);
+	bool is_query_url(const std::string& url);
+	bool is_filter_url(const std::string& url);
+	bool is_exec_url(const std::string& url);
 
-	static std::string get_content(xmlNode* node);
-	static std::string get_basename(const std::string& url);
+	std::string get_content(xmlNode* node);
+	std::string get_basename(const std::string& url);
 
-	static std::string escape_url(const std::string& url);
-	static std::string unescape_url(const std::string& url);
-	static void initialize_ssl_implementation(void);
+	std::string escape_url(const std::string& url);
+	std::string unescape_url(const std::string& url);
+	void initialize_ssl_implementation(void);
 
-	static unsigned int gentabs(const std::string& str);
+	unsigned int gentabs(const std::string& str);
 
-	static int mkdir_parents(const std::string& pathname,
+	int mkdir_parents(const std::string& pathname,
 		mode_t mode = 0755);
 
-	static std::string make_title(const std::string& url);
+	std::string make_title(const std::string& url);
 
-	static int run_interactively(const std::string& command,
+	int run_interactively(const std::string& command,
 		const std::string& caller);
 
-	static std::string getcwd();
+	std::string getcwd();
 
-	static int strnaturalcmp(const std::string& a, const std::string& b);
+	int strnaturalcmp(const std::string& a, const std::string& b);
 
-	static void remove_soft_hyphens(std::string& text);
+	void remove_soft_hyphens(std::string& text);
 
-	static bool is_valid_podcast_type(const std::string& mimetype);
+	bool is_valid_podcast_type(const std::string& mimetype);
 
-	static std::string get_default_browser();
+	std::string get_default_browser();
 
-private:
-	static void append_escapes(std::string& str, char c);
-};
-
-class ScopeMeasure {
-public:
-	ScopeMeasure(const std::string& func, Level ll = Level::DEBUG);
-	~ScopeMeasure();
-	void stopover(const std::string& son = "");
-
-private:
-	struct timeval tv1, tv2;
-	std::string funcname;
-	Level lvl;
-};
+	void append_escapes(std::string& str, char c);
```

Comment:
Since this function used to be private, please remove it from the header.

Reply:
makes sense.